src/core.js

```diff
@@ -272,6 +272,16 @@ $.extend( $.validator, {
 				this.element( element );
 			}
 		},
+		oninput: function( element ) {
+			if ( element.name in this.submitted || element.name in this.invalid ) {
+				this.element( element );
+			} else {
+				return;
+			}
+		},
+
+		// DEPRECATED
+		// Please use `oninput` instead.
 		onkeyup: function( element, event ) {
```

Comment:
But as you know, removing `onkeyup` is a BC break and should be done in version `2.0.0`